chartfx-math/src/test/java/de/gsi/math/spectra/ShortTimeFourierTransformTest.java

```diff
@@ -71,77 +70,67 @@ public void testRealSTFT() {
         assertEquals(sine.getAxisDescription(DIM_Y).getUnit(), sineSpectrogram.getAxisDescription(DIM_Z).getUnit());
 
         // check spectrogram
-        assertArrayEquals(new double[] { 0, 1 * step * dt, 2 * step * dt, 3 * step * dt, 4 * step * dt, 5 * step * dt },
-                sineSpectrogram.getValues(DIM_Y), dt);
+        assertArrayEquals(new double[] { 0, 1 * step * dt, 2 * step * dt, 3 * step * dt, 4 * step * dt, 5 * step * dt }, sineSpectrogram.getGridValues(DIM_Y), dt);
         // get spectrogram raw data
         final double[][] zValues = getDataAsNestedArray(sineSpectrogram);
         final DoubleFFT_1D fastFourierTrafo = new DoubleFFT_1D(nFft);
         // compare to transform t = 0
-        final double[] vals1ref = Arrays.copyOfRange(sine.getYValues(), 0, nFft);
+        final double[] vals1ref = Arrays.copyOfRange(sine.getValues(DataSet.DIM_Y), 0, nFft);
         Apodization.Hann.apodize(vals1ref);
         fastFourierTrafo.realForward(vals1ref);
         final double[] vals1refMag = SpectrumTools.computeMagnitudeSpectrum(vals1ref, true);
         assertArrayEquals(vals1refMag, zValues[0]);
         // compare to transform t = 2 * step * dt
-        final double[] vals2ref = Arrays.copyOfRange(sine.getYValues(), 2 * step, 2 * step + nFft);
+        final double[] vals2ref = Arrays.copyOfRange(sine.getValues(DataSet.DIM_Y), 2 * step, 2 * step + nFft);
         Apodization.Hann.apodize(vals2ref);
         fastFourierTrafo.realForward(vals2ref);
         final double[] vals2refMag = SpectrumTools.computeMagnitudeSpectrum(vals2ref, true);
         assertArrayEquals(vals2refMag, zValues[2]);
         // compare to transform t = 5*step*dt
         final double[] vals3ref = new double[nFft];
-        System.arraycopy(sine.getYValues(), 5 * step, vals3ref, 0, nFft - 32 - step);
+        System.arraycopy(sine.getValues(DataSet.DIM_Y), 5 * step, vals3ref, 0, nFft - 32 - step);
         Apodization.Hann.apodize(vals3ref);
         fastFourierTrafo.realForward(vals3ref);
         final double[] vals3refMag = SpectrumTools.computeMagnitudeSpectrum(vals3ref, true);
         assertArrayEquals(vals3refMag, zValues[5]);
 
         // compare to transform t = 5*step*dt with ZOH
-        ShortTimeFourierTransform.real(sine, sineSpectrogram, nFft, step, Apodization.Hamming, Padding.ZOH, false,
-                true);
+        ShortTimeFourierTransform.real(sine, sineSpectrogram, nFft, step, Apodization.Hamming, Padding.ZOH, false, true);
         final double[][] zValuesZOH = getDataAsNestedArray(sineSpectrogram);
         final double[] vals3refZOH = new double[nFft];
-        System.arraycopy(sine.getYValues(), 5 * step, vals3refZOH, 0, nFft - 32 - step);
+        System.arraycopy(sine.getValues(DataSet.DIM_Y), 5 * step, vals3refZOH, 0, nFft - 32 - step);
```

Comment:
Codacy found an issue: [Unnecessary use of fully qualified name 'DataSet.DIM_Y' due to existing static import 'de.gsi.dataset.DataSet.DIM_Y'](https://app.codacy.com/manual/GSI/chart-fx/pullRequest?prid=5780327)